Mathlib/Analysis/Asymptotics/TVS.lean

```diff
@@ -193,7 +193,7 @@ lemma IsLittleOTVS.insert [TopologicalSpace α] {x : α} {s : Set α}
 lemma IsLittleOTVS.bot : f =o[𝕜;⊥] g :=
   fun u hU => ⟨univ, by simp⟩
 
-theorem IsLittleOTVS.add [IsTopologicalAddGroup E] [ContinuousSMul 𝕜 E]
+theorem IsLittleOTVS.add [ContinuousAdd E] [ContinuousSMul 𝕜 E]
```

Comment:
This is one of those spurious generalizations. A group with `ContinuousAdd` and `ContinuouSMul` is a `IsTopologicalAddGroup`.